docs/apis/storm_compatibility.md

```diff
@@ -201,6 +201,26 @@ DataStream<BoltOutputType> s2 = splitStream.select("s2").transform(/* use Bolt f
 
 See [SpoutSplitExample.java](https://github.com/apache/flink/tree/master/flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/split/SpoutSplitExample.java) for a full example.
 
+## Configure for embedded Spouts/Bolts
+Embedder spouts/bolts can be configure with user defined parameters. User defined parameters is a config `Map` and be called by `Spout.open(...)` and `Bolt.prepare()`
+as first parameter. Configuration can be used in storm topologies mode or flink mode.
+
+ 1.Storm topologies mode example
```

Comment:
I don't think we need example code for embedded mode. (See "multiple output streams":  If a whole topology is executed using FlinkTopologyBuilder etc., there is no special attention required – it works as in regular Storm. ). A simple sentence in the paragraph should be sufficient.